mmstudio/src/main/java/org/micromanager/internal/utils/PrioritizedEventBus.java

```diff
@@ -29,7 +45,11 @@ public void register(Object o) {
    public void register(Object o, Integer priority) {
       EventBus subBus;
       if (!prioritizedBuses_.containsKey(priority)) {
-         subBus = new EventBus(EventBusExceptionLogger.getInstance());
+         if (async_) {
+            subBus = new AsyncEventBus(executorService_, EventBusExceptionLogger.getInstance());
```

Comment:
(Mostly from curiosity), did it turn out that async event bus actually improves performance?

Reply:
My testing was not intensive, but it did not seem to make much of a difference.  Decided to do it asynchronous in any case, as this event is public and plugin code consuming the event may want to take its time.

Reply:
Sounds good.